### PR TITLE
Improve the page.js

### PR DESCRIPTION
**Pull request by 12345rams**
Proposed Changes:
This pull request aims to improve the layout of NFT cards to enhance the user experience and adapt to different screen sizes. The issue reported that when multiple NFT cards are displayed, they are aligned vertically, which is not the most efficient use of screen space. The expected improvement is to introduce a responsive layout that aligns cards differently based on the screen size.
Changes Made:
For larger screens (desktop), a grid layout is implemented. Three NFT cards are displayed in one row, and subsequent cards continue in the next row.
For mobile view, the cards maintain their current vertical alignment to fit within the limited screen width, ensuring readability and usability on smaller screens.

### CODE REVIEW

File: client/app/Explore/page.js [hunk 1]
```diff
@@ -51,82 +51,56 @@ const Explore = () => {
     fetchNFTData();
   }, [nftTransfers]);
 
-  // console.log(nftDataforcard);
-
-  if (loading) {
```
Comment (ojasaklechayt) [inline client/app/Explore/page.js:56]: Please keep this loading page as it is because it will improve the UX for the user.

**Comment by ojasaklechayt** [comment]
@12345rams Thankyou for the PR but before going further. 

- Please message in the issue you are working on. 
- Do not make PR without you getting the issue assigned. 
- Please mention the issue number in the pr description by using # ( issue number ). 

Also there's one I would like you to improve is that - 

image

The width of the card is very high. Please do change it and make it align to the ones in the collection page.

**Comment by 12345rams** [comment]
I'm Creating a new pull request with include all changes witch you want.

**Comment by ojasaklechayt** [comment]
Ok

**Pull request closed without merging**